File: php/Disk.php
<?php

class Disk
{
    private $type = "";
    private $price = 0;
    private $capacity = 0;

    # Constructor
    public function __construct($t="", $c=0, $p=0)
    {
        $this->type = $t;
        $this->price = $p;
        $this->capacity = $c;
    }

    # Setter and Getter
    public function setType($ty="")
    {
        $this->type = $ty;
    }

    public function getType()
    {
        return $this->type;
    }

    public function setPrice($p=0)
    {
        $this->price = $p;
    }

    public function getPrice()
    {
        return $this->price;
    }

    public function setCapacity($c=0)
    {
        $this->capacity = $c;
    }

    public function getCapacity()
    {
        return $this->capacity;
    }

    # Menampilkan isi kelas
    public function showDisk()
    {
        echo "Disk Type       : " . $this->type . "<br>";
        echo "Disk Capacity   : " . $this->capacity . " Gb<br>";
        echo "Disk Price      : Rp. " . $this->price . "<br>";
    }
}

?>

File: python/Disk.py
class Disk():

    # Constructor
    def __init__(self):
        self.type = ""
        self.capacity = 0
        self.price = 0
    
    # Setter and Getter
    def setType(self, type):
        self.type = type
    
    def getType(self):
        return self.type

    def setCapacity(self, capacity):
        self.capacity = capacity

    def getCapacity(self):
        return self.capacity
    
    def setPrice(self, price):
        self.price = price
    
    def getPrice(self):
        return self.price
    
    # Menampilkan isi Kelas
    def showDisk(self):
        print("Disk Type       : " + str(self.type))
        print("Disk Capacity   : " + str(self.capacity) + " Gb")
        print("Disk Price      : Rp. " + str(self.price))